src/components/header-search-input/header-search-input.module.css

```diff
@@ -31,6 +31,12 @@
   }
 }
 
+.popoverContainer {
+  align-items: center;
+  display: flex;
+  height: 100%;
+}
```

Comment:
I chose to use this approach rather than modify styles to `Popover` that way the changes were isolated. I think these styles would have negatively affected where we use `Popover` in other places.